src/actions/attributes.ts

```diff
@@ -57,13 +57,13 @@ export function set_dataset_attribute(this: StreamElement) {
 }
 
 export function set_property(this: StreamElement) {
-  const name = this.getAttribute("name")
-  const value = this.getAttribute("value") || ""
+  const property = this.getAttribute("property")
```

Comment:
This is technically a breaking change, but I think this should be fine at the current stage of the project

Reply:
If one is using `turbo_power-rails` then it is a fix but otherwise, yes, a breaking change.

Let me know which direction you want to go. It is not a big deal to revert back to the `name` attribute. In that case I'll make a PR for `turbo_power-rails`.

Reply:
I'd probably keep it at `name`, not because of the backwards compatibility, but more because so that it looks and feels similar to the other actions.